binr/r2r/run.c

```diff
@@ -1203,6 +1203,19 @@ R_API R2RTestResultInfo *r2r_run_test(R2RRunConfig *config, R2RTest *test) {
 	}
 	}
 	bool broken = r2r_test_broken (test);
+#if ASAN
+# if !R2_ASSERT_STDOUT
```

Comment:
why is this test required? What happens if R2_ASSERT_STDOUT is not defined?

Reply:
Assertion output goes to stderr which then invalidates an assumption made by a line further down:

```c
	        (!strstr (out->out, "WARNING:") && !strstr (out->out, "ERROR:") && !strstr (out->out, "FATAL:")))
```